src/shared/components/plots/StackedBarPlot.tsx

```diff
@@ -215,6 +216,9 @@ export default class StackedBarPlot extends React.Component<IStackedBarPlotProps
     }
 
     @computed get maxMajorCount() {
+        if(!!this.props.percentage){
```

Comment:
this is a nitpick but since percentage is boolean, !!percentage will be true iff percentage is true. so you can just check `if(this.props.percentage)`